src/lib/SearchModal.svelte

```diff
@@ -1,70 +1,45 @@
 <script>
-	import { searchBarOpen } from './utils.js';
-  import { createEventDispatcher, onMount } from 'svelte';
+  import { searchBarOpen } from './utils.js';
+  import { onMount } from 'svelte';
   import { fade } from 'svelte/transition';
 
-  // Props
-  // export let searchBarOpen = $searchBarOpen;
-  export let searchTerm = '';
-
   // State management
   let searchResults = [];
   let loading = false;
   let error = null;
+  let term = '';
 
-  // Event dispatcher
-  // const dispatch = createEventDispatcher();
-
-  // Search function (async to simulate API call)
-  async function searchProjects(term) {
+  async function searchProjects() {
     if (!term) {
       searchResults = [];
       return;
     }
 
     loading = true;
     try {
-      // Simulated search - replace with actual API call
-      const mockProjects = [
-        { id: 1, name: 'Digital Health Initiative', description: 'Improving healthcare access' },
-        { id: 2, name: 'Education Technology Platform', description: 'Online learning solutions' },
-        { id: 3, name: 'Climate Action Tracker', description: 'Environmental monitoring system' }
-      ];
+      const response = await fetch(`/api/projects?term=${term}&page=1&limit=10`, {
+        method: 'GET',
```

Comment:
since it's a `GET` request, we could leave out the 
```js
{ 
    method: "GET",
    headers: { "Content-Type": "application/json" }
}
```
but this works too.

Reply:
Works, but now i'll have to urge to remove it in all GET requests lol